test/user-profile/components/AuthApplicationSection.unit.spec.jsx

```diff
@@ -1,36 +1,26 @@
 import React from 'react';
-// import { findDOMNode } from 'react-dom';
-// import ReactTestUtils from 'react-dom/test-utils';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
-// import sinon from 'sinon';
 
 import AuthApplicationSection from '../../../src/js/user-profile/components/AuthApplicationSection.jsx';
-// import handleVerify from '../../../../src/js/common/helpers/login-helpers.js';
 
 describe('<AuthApplicationSection>', () => {
   const props = {
     userProfile: {},
     verifyUrl: 'http://fake-verify-url'
   };
-
   it('should render', () => {
     const tree = SkinDeep.shallowRender(<AuthApplicationSection {...props}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.not.be.undefined;
   });
-
-  // TODO: figure out how to get spy to be called
-  /*
-  it('should call handler when verify link is clicked', () => {
-    const verifySpy = sinon.spy(handleVerify);
-    const section = ReactTestUtils.renderIntoDocument(
-      <AuthApplicationSection {...props }/>
-    );
-    ReactTestUtils.Simulate.click(
-      findDOMNode(section).querySelector("a[href='#']"));
-    expect(verifySpy.called.with(props.verifyUrl)).to.be.true;
```

Comment:
Hm, looking at this a bit, I assume what we were trying to do here is test that `handleVerify` actually gets called when this link is clicked. While this would be a nice to have, seems like it was difficult to figure out how to get the spy actually working. If it's not too much trouble it seems useful to keep the commented out code in for now. We should, however, come up with a plan for how to revisit these kinds of things, there are lots of commented out tests right now :/ curious to hear what @raquelromano-gov thinks too though!